lib/c-api/src/wasm_c_api/types/function.rs

```diff
@@ -97,7 +96,6 @@ pub unsafe extern "C" fn wasm_functype_params(ft: &wasm_functype_t) -> *const wa
     out as *const _
 }
 
-// TODO: fix memory leak
```

Comment:
For those deletions, take a look at https://github.com/wasmerio/wasmer/pull/1795/commits/8ae283507e4912ef9d56a11f18e3edde867825b0.